### PR TITLE
Validate jupyter notebooks with pytest and nbval [ON HOLD]

### LINKED ISSUE #13: Evaluate notebooks as a part of testing suite
Relevant tools:
- https://github.com/computationalmodelling/nbval
- https://github.com/marella/nbimport
- https://github.com/spatialaudio/nbsphinx
- https://github.com/jupyter/nbconvert
- https://github.com/kynan/nbstripout
- https://github.com/marketplace/review-notebook-app
- https://github.com/ngoldbaum/RunNotebook

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Closes #13

### CODE REVIEW

File: .github/workflows/eval-notebooks.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```
Comment (pearu) [inline .github/workflows/eval-notebooks.yml:11]: What does it mean that the workflow is triggered only for the main branch? When creating a PR that includes a notebook change, will the notebook be evaluated?

**Comment by guilhermeleobas** [comment]
Failure is related to ibis not shipping with impala. I can't reproduce it locally and it ibis conda-forge recipe depends on impala:
https://github.com/conda-forge/ibis-framework-feedstock/blob/7ad393704a340e3af48d18773e1c6d41e700e2ae/recipe/meta.yaml#L30

File: .github/workflows/eval-notebooks.yml (hunk 1 above)
Comment (pearu) [reply .github/workflows/eval-notebooks.yml:11]: Got an affirmative answer.

**Comment by pearu** [comment]
IIUC, the error message is `ibis.{name} requires impala backend to be installed`. Could we add `impala` to the environment where nb evaluator is run?

**Comment by guilhermeleobas** [comment]
@pearu @brenocfg we can finally land this!

File: .github/workflows/eval-notebooks.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          mamba-version: "*"
+          channels: conda-forge
+          environment-file: .conda/environment.yml
+
+      - name: Install additional packages
+        shell: bash -l {0}
+        run: |
+          mamba install omniscidb -c conda-forge
+          pip install nbval ibis-omniscidb matplotlib
+          pip uninstall rbc-project -y
```
[the remark this answers is not in the document]
Comment (pearu) [reply .github/workflows/eval-notebooks.yml:37]: Not sure if this is needed: `python setup.py develop` should override installed versions of rbc.

**Pull request closed without merging**